### PR TITLE
Lab1 Implemented Fibonacci sequence

### LINKED ISSUE #367: Lab1 `implemented `fibonacci sequence

### PR DESCRIPTION
**Pull request by sirigithub**
Fixes #367 

Review of colleague's PR #

Changes proposed in this PR:

### CODE REVIEW

**Review by n0npax — DISMISSED** [review]
lgmt

File: 01_fib/sirigithub/main.go [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+	for i := 1; i <= absVal; i++ {
+		sign := 1
+
+		if n < 0 && (i%2 == 0) { // for negative fib input, print a negative sign on even index
+			sign = -1
+		}
+		fmt.Fprintln(out, current*sign)
+		previous, current = current, (current + previous)
+	}
```
Comment (juliaogris) [inline 01_fib/sirigithub/main.go:29]: functionally correct. For completeness a slightly shorter version:
```go
	sign := 1
	if n < 0 {
		sign = -1
	}

	n1, n2 := 0, 1
	for j := 0; j < n*sign; j++ {
		n1, n2 = n2, n1+sign*n2
		fmt.Fprintln(out, n1)
	}
```

File: 01_fib/sirigithub/main.go [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+}
+
+func main() {
+
+	fib(7)
+
```
Comment (juliaogris) [inline 01_fib/sirigithub/main.go]: ditto

File: 01_fib/sirigithub/main.go [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+		if n < 0 && (i%2 == 0) { // for negative fib input, print a negative sign on even index
+			sign = -1
+		}
+		fmt.Fprintln(out, current*sign)
+		previous, current = current, (current + previous)
+	}
+}
+
+func main() {
+
```
Comment (juliaogris) [inline 01_fib/sirigithub/main.go]: remove unnecessary blank line

**Review by juliaogris — CHANGES_REQUESTED** [review]
Please read through the [Collected Code Review Comments](https://github.com/anz-bank/go-course/wiki/Lab-1:-Collected-Code-Review-Comments)
- [Fix your pr description](https://github.com/anz-bank/go-course/wiki/Lab-1:-Collected-Code-Review-Comments#pr-description)
- [Fix commit messages](https://github.com/anz-bank/go-course/wiki/Lab-1:-Collected-Code-Review-Comments#commit-messages)

**Pull request closed without merging**